mcpwm_foc.c

```diff
@@ -2818,15 +2820,15 @@ void mcpwm_foc_adc_int_handler(void *p, uint32_t flags) {
 					motor_now->m_motor_state.phase = motor_now->m_phase_now_encoder_no_index;
 				}
 
-				if (!motor_now->m_phase_override) {
+				if (!motor_now->m_phase_override && !(motor_now->m_control_mode == CONTROL_MODE_OPENLOOP_PHASE)) {
```

Comment:
Any reason why `!(motor_now->m_control_mode == CONTROL_MODE_OPENLOOP_PHASE)` instead of `motor_now->m_control_mode != CONTROL_MODE_OPENLOOP_PHASE`?